Automattic/wp-calypso
# Fix and re-enable Site Editor Tracking tests for global styles update/save events

#### Changes proposed in this Pull Request

A Follow up to the test skips we introduced in https://github.com/Automattic/wp-calypso/pull/56544

* Re-enables global styles update/save tracks tests by updating them for the new interface and removing the skips. 
* Cleanup - moves global styles interface helper functions into the site editor component and out of the test file.

#### Testing instructions



* Run the test file altered in this PR (`wp-calypso-gutenberg-site-editor-tracking-spec`) with the `GUTENBERG_EDGE=true` (for v11.8 or higher) flag and verify it passes.



Related to https://github.com/Automattic/wp-calypso/issues/56540
